src/coreclr/jit/instrsxarch.h

```diff
@@ -640,6 +645,22 @@ INST3(vinserti32x8,      "inserti32x8",     IUM_WR, BAD_CODE,     BAD_CODE,
 INST3(vpmovd2m,          "pmovd2m",         IUM_WR, BAD_CODE,     BAD_CODE,     PACK4(0xF3, 0x0F, 0x38, 0x39),           INS_TT_NONE,                         Input_32Bit    | REX_W0_EVEX                  | Encoding_EVEX)
 INST3(vpmovq2m,          "pmovq2m",         IUM_WR, BAD_CODE,     BAD_CODE,     PACK4(0xF3, 0x0F, 0x38, 0x39),           INS_TT_NONE,                         Input_64Bit    | REX_W1_EVEX                  | Encoding_EVEX)
 
+INST3(vpmovm2b,         "vpmovm2b",          IUM_WR, BAD_CODE,     BAD_CODE,     PACK4(0xF3, 0x0F, 0x38, 0x28),          INS_TT_NONE,                         Input_8Bit     | REX_W0_EVEX                  | Encoding_EVEX ) 
+INST3(vpmovm2w,         "vpmovm2w",          IUM_WR, BAD_CODE,     BAD_CODE,     PACK4(0xF3, 0x0F, 0x38, 0x28),          INS_TT_NONE,                         Input_16Bit    | REX_W1_EVEX                  | Encoding_EVEX ) 
+INST3(vpmovm2d,         "vpmovm2d",          IUM_WR, BAD_CODE,     BAD_CODE,     PACK4(0xF3, 0x0F, 0x38, 0x38),          INS_TT_NONE,                         Input_32Bit    | REX_W0_EVEX                  | Encoding_EVEX ) 
+INST3(vpmovm2q,         "vpmovm2q",          IUM_WR, BAD_CODE,     BAD_CODE,     PACK4(0xF3, 0x0F, 0x38, 0x38),          INS_TT_NONE,                         Input_64Bit    | REX_W1_EVEX                  | Encoding_EVEX ) 
```

Comment:
nit: There is still per ISA groupings and they are alphabetized within those groups.

`vpmovm2b` and `vpmovm2w` should be under `AVX512BW`
`vpmovm2d` and `vpmovm2q` should be under `AVX512DQ`

Keeping this consistency will make it much easier to add asserts or additional validation later that we aren't emitting incorrect instructions.

Reply:
Same with `vpcmp*` and `vpcmpu*` below

Reply:
Okay. I assume we do not the need the "v" in front of the string for display
`INST3(vpmovm2b,         "pmovm2b",          IUM_WR..`

instead of

`INST3(vpmovm2b,         "vpmovm2b",          IUM_WR`

Reply:
Correct

Reply:
Done